e2e/playwright.config.ts

```diff
@@ -14,7 +14,7 @@ dotenv.config();
 const config: PlaywrightTestConfig = {
     testDir: './tests',
     /* Maximum time one test can run for. */
-    timeout: 180000, // some of the publication flow and coauthor tests exceed 2 minutes. We should try and streamline them but for now set this to 3 mins
+    timeout: 300000, // some of the publication flow and coauthor tests exceed 2 minutes. We should try and streamline them but for now set this to 3 mins
```

Comment:
5 minutes for a test 😨

Reply:
It's bad :( but we have an item this sprint to begin streamlining these tests so that they are more reliable and quick to run (OC-764).